pkg/container/docker_network.go

```diff
@@ -14,10 +15,11 @@ func NewDockerNetworkCreateExecutor(name string) common.Executor {
 			return err
 		}
 
-		_, err = cli.NetworkCreate(ctx, name, types.NetworkCreate{})
+		network, err := cli.NetworkCreate(ctx, name, types.NetworkCreate{})
 		if err != nil {
 			return err
 		}
+		fmt.Printf("%#v", network.ID)
```

Comment:
Please use `log` instead of `fmt`, we should limit `fmt` only to formatting strings

Reply:
Oh sorry, it was a rogue line, my apologies